packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -104,29 +112,25 @@ export interface EditableTree extends Iterable<EditableField> {
      */
     readonly [proxyTargetSymbol]: object;
 
-    /**
-     * Anchor to this node.
-     * Valid as long as this EditableTree's context is not freed.
-     * Might not point to any node if this node is deleted from the document.
-     *
-     * TODO: When a proper editing API is exposed on EditableTree directly,
-     * this should become an implementation detail and rbe removed from this API surface.
-     */
-    readonly [anchorSymbol]: Anchor;
-
     /**
      * Gets the field of this node by its key without unwrapping.
      */
-    [getWithoutUnwrappingSymbol](fieldKey: FieldKey): EditableField;
+    [getField](fieldKey: FieldKey): EditableField;
 
     /**
      * Fields of this node, indexed by their field keys.
      *
      * This API exposes content in a way depending on the {@link Multiplicity} of the {@link FieldKind}.
      * Sequences (including empty ones) are always exposed as {@link EditableField}s,
      * and everything else is either a single EditableTree or undefined depending on if it's empty.
+     *
+     * It is possible to use this indexed access to delete the field using the `delete` operator and
+     * to set the value of the field or, more precisely, of its existing node using the simple assignment operator (`=`)
+     * if the field is defined as `optional` or `value`, its node {@link isPrimitive} and the value is a {@link PrimitiveValue}.
+     * Concurrently setting the value will follow the "last-write-wins" semantics.
```

Comment:
I'm curious what you think of allowing field to be set with a single node when the field is an optional or value field.
It doesn't have to be in this PR but it seems like it would make the API more straightforward.